openstack/compute/v2/extensions/floatingip/results.go

```diff
@@ -0,0 +1,99 @@
+package floatingip
+
+import (
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// A FloatingIP is an IP that can be associated with an instance
+type FloatingIP struct {
+	// ID is a unique ID of the Floating IP
+	ID int `mapstructure:"id"`
```

Comment:
I'll give it a shot :)

The JSON parsing is rather strict, right? But mapstructure is more flexible?